content/en/blog/2023/euwg-future-of-observability-panel/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title:
+  'OTel End User Working Group Presents: The Future of Observability Panel
+  Discussion'
+linkTitle: The Future of Observability Panel
+date: 2023-10-17
+author: >-
+  [Adriana Villela](https://github.com/avillela) (Lightstep)
+cSpell:ignore: Dyrmishi Farfetch Iris Mellifera Nočnica Samuel Vijay youtube
+---
+
+![The Evolution of Observability Practices, featuring Guests David Wynn, Austin Parker, Vijay Samuel, Nočnica Mellifera, and Iris Dyrmishi](evolution-o11y-practices-social-card.jpeg)
+
+OpenTelemetry has greatly impacted the Observability landscape over the past few
+years. While it introduces an open standard for telemetry generation and
+collection, what is actually improving the lives of Observability teams?
+
+Guests David Wynn, Austin Parker, Vijay Samuel, Nočnica Mellifera, and Iris
+Dyrmishi came together to share their experiences and insights on the evolution
```

Comment:
do you want to link everyone's github/linkedin/... profile?

Reply:
Yes, that's a great idea @svrnm. Added!